src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -508,39 +480,48 @@ class CosmasGoogleCloudService(
                             responseObserver: StreamObserver<CosmasProto.DeleteFileResponse>) = logging(
             "deleteFile", request.info.projectId, request.fileId,
             other = mapOf("fileName" to request.fileName)) {
-        val cemeteryName = "${request.info.projectId}-cemetery"
-        val cemeteryBytes: Blob? = try {
-            this.storage.get(getBlobId(cemeteryName, request.info))
+        try {
+            val fileToDelete = DeletedFileInfo.newBuilder()
+                    .setFileId(request.fileId)
+                    .setFileName(request.fileName)
+                    .build()
+            deleteFiles(listOf(fileToDelete), request.removalTimestamp, request.info)
         } catch (e: StorageException) {
             handleStorageException(e, responseObserver)
             return@logging
         }
-        val newTomb = FileTomb.newBuilder()
-                .setFileId(request.fileId)
-                .setFileName(request.fileName)
-                .setRemovalTimestamp(request.removalTimestamp)
-                .build()
+
+        val response = DeleteFileResponse.newBuilder().build()
+        responseObserver.onNext(response)
+        responseObserver.onCompleted()
+    }
+
+    private fun deleteFiles(filesToDelete: List<DeletedFileInfo>, removalTimestamp: Long, info: ProjectInfo) {
+        val cemeteryName = "${info.projectId}-cemetery"
+
+        val cemeteryBytes: Blob? = this.storage.get(getBlobId(cemeteryName, info))
         val cemetery = if (cemeteryBytes == null) {
             FileCemetery.newBuilder()
         } else {
             FileCemetery.parseFrom(cemeteryBytes.getContent()).toBuilder()
         }
-        val curTime = clock.millis()
-        val ttl = getTTL(request.info)
-        val tombs = cemetery.cemeteryList.toMutableList()
-        tombs.removeIf { it.removalTimestamp + ttl > curTime }
-        try {
-            this.storage.create(
-                    getBlobInfo(cemeteryName, request.info),
-                    cemetery.clearCemetery().addAllCemetery(tombs).addCemetery(newTomb).build().toByteArray())
-        } catch (e: StorageException) {
-            handleStorageException(e, responseObserver)
-            return@logging
+        for (file in filesToDelete) {
+            val newTomb = FileTomb.newBuilder()
+                    .setFileId(file.fileId)
+                    .setFileName(file.fileName)
+                    .setRemovalTimestamp(removalTimestamp)
+                    .build()
+            cemetery.addCemetery(newTomb)
+            LOG.info("File={} with name={} has been added to cemetery", file.fileId, file.fileName)
```

Comment:
debug?

Reply:
I think it's usefull to log it anyway, so it's info like everywhere.